### PR TITLE
Fix: punctuations in zh langauge file

### PR DESCRIPTION
**Pull request by WenSimEHRP**
The current langauge strings does not follow the format of standard (simplified) Chinese, and needs to be fixed. I corrected the strings using a program but I'm not very sure if it touches the variables or something. Since I am new to translating TheoTown, I hope that somebody could help me check if I missed or broke anything. Thanks

### CODE REVIEW

**Comment by WenSimEHRP** [comment]
It's been two months and I haven't received any comments from a dev or others while other PRs were merged. I hope there is a specific reason for that.

By the way this is a systematic fix and does not add new translations or change current translations, so merging it is basically fine.

File: values-zh/strings.xml [hunk 1]
```diff
@@ -1,3435 +1,3435 @@
...
+    <string name="draft_landfill00_title">垃圾处理厂</string>
+    <string name="draft_mltry_depot00_title">仓库</string>
+    <string name="draft_mltry_rocketlauncher00_title">火箭发射装置</string>
+    <string name="draft_mltry_road00_title">军用公路</string>
+    <string name="draft_firedepartment02_title">主消防站</string>
+    <string name="draft_wasteincinerator00_title">垃圾焚烧炉</string>
+    <string name="rank_title">最新城市等级</string>
+    <string name="rank_reward">你得到 %1$s 作为奖励</string>
+    <string name="rank_newrank">恭喜，%1$s 达到了新等级：</string>
+    <string name="ra
```
Comment (JustAnyones) [inline values-zh/strings.xml:2334]: @LobbyDivinus

**Review by JustAnyones — COMMENTED** [review]
I checked the rest, next time I will reject pull requests that seemingly replace a dozen lines without a good reason.

**Comment by WenSimEHRP** [comment]
> I checked the rest, next time I will reject pull requests that seemingly replace a dozen lines without a good reason.

I assume that "systematic change of punctuations" qualifies as good reason.
By the way git doesn't seem to like crlf line ending, which results it reporting all lines are changed. Sorry if that slowed the checking.

**Pull request merged**